hadoop-hdfs-project/hadoop-hdfs-native-client/src/main/native/libhdfspp/lib/common/async_stream.h

```diff
@@ -41,7 +41,6 @@ typedef boost::asio::const_buffers_1   ConstBuffer;
 class AsyncStream  {
 public:
   using executor_type = boost::asio::system_executor;
```

Comment:
Could you try making this type alias private as well?

Reply:
I tried that, but it seems to be referred by block_reader and datatransfer in lib/reader

Reply:
Ok. Thanks Suraj. You may resolve my comment.